File: Java/Day 7/TaskNine.java
public class TaskNine{
	public static void main(String[] args){
		int multiple = 1;
		int sumOfMultiples = 0;
		for (int count = 0; count < 5; count++){
			multiple *= 4;
			sumOfMultiples += multiple;
		} 

		multiple = 1;

		for (int count = 0; count < 5; count++){
			multiple *= 8;
			sumOfMultiples += multiple;
		} 
		System.out.println(sumOfMultiples*sumOfMultiples);
	}
}

File: JavaScript/Day 7/TaskNine.js
let multiple = 1
let sumMultiples = 0
for (let count = 0; count < 5; count++){
	multiple *= 4;
	sumMultiples += multiple
} 

multiple = 1

for (let count = 0; count < 5; count++){
	multiple *= 8;
	sumMultiples += multiple
}
console.log(sumMultiples**2);